src/components/expired-subscription-modal/index.jsx

```diff
@@ -31,6 +46,7 @@ const ExpiredSubscriptionModal = () => {
         ),
       }}
       />
+      <p>{userEmail}{enterpriseCustomer.uuid}</p>
```

Comment:
[clarification] Is it intentional to expose the `enterpriseCustomer.uuid` (and user's email) in the modal body contents? Seems a bit odd, IMHO. Why would the learner want/need to see their customer's UUID?

Reply:
Oooh my bad, I added it while testing and forgot to remove it while committing changes. Thanks for pointing this out. [Adam the savior] 👍